postgresql/resource_postgresql_grant_test.go

```diff
@@ -2,12 +2,130 @@ package postgresql
 
 import (
 	"fmt"
+	"reflect"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/lib/pq"
 )
 
+func TestCreateGrantQuery(t *testing.T) {
+	var databaseName string = "foo"
+	var roleName string = "bar"
```

Comment:
```suggestion
	var databaseName = "foo"
	var roleName = "bar"
```
You should omit type declaration as it's inferred from the value (and it makes linter unhappy :) )